azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -154,6 +159,28 @@ public ExecutorManager(final Props azkProps, final ExecutorLoader loader,
     this.cleanerThread = new CleanerThread(executionLogsRetentionMs);
     this.cleanerThread.start();
 
+    this.globalProps = setupGlobalProps(azkProps);
+  }
+
+  private Props setupGlobalProps(Props azkProps) {
+    String globalPropsPath = azkProps.getString("executor.global.properties", null);
+    if (Objects.nonNull(globalPropsPath)) {
+      try {
+        if (globalPropsPath.startsWith("/")) {
+          return new Props(null, new File(globalPropsPath));
+        } else {
+          final URL globalPropsUrl = Resources.getResource(globalPropsPath);
```

Comment:
Do we need to introduce the new `Resources` dependency? Can we accomplish the same thing with the the Java `File` library?